bot.py

```diff
@@ -34,6 +35,93 @@
 
 bot = commands.Bot(command_prefix='!')
 
+#variables needed for !uwu
+#I mean what else would it be for?
+suffixes = [
```

Comment:
We are currently using a json file as a database. What are your thoughts on dumping suffixes into a "suffix" object of our json library for consistency?